client/my-sites/plugins/plugin-ratings/style.scss

```diff
@@ -17,40 +17,39 @@
 	}
 }
 
-.plugin-ratings__rating-tier {
-	line-height: 18px;
-}
-
 .plugin-ratings__rating-text {
-	font-size: 11px;
+	font-size: 12px;
```

Comment:
because I got rid of the 'uppercase' css dec. We don't really use lowercase 11px much, but we seem to use 12 more—more readable, less microscopic.